Client/Ecosia/UI/MyImpact/EcosiaHome.swift

```diff
@@ -204,6 +204,11 @@ final class EcosiaHome: UICollectionViewController, UICollectionViewDelegateFlow
         super.viewWillAppear(animated)
         news.load(session: .shared, force: items.isEmpty)
         Analytics.shared.navigation(.view, label: .home)
+
+        if Referrals.isActive {
+            referrals.refresh(force: true)
```

Comment:
@ecotopian we are always forcing a refresh?

Reply:
@vauxhall no we don't. In FirefoxHomeViewController (NTP) on viewWillAppear we do a refresh without force